packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -252,6 +261,7 @@ export function makeRumPublicApi(
 
     startSessionReplayRecording: monitor(recorderApi.start),
     stopSessionReplayRecording: monitor(recorderApi.stop),
+    getReplayLink: monitor((subdomain = 'app') => getReplayLinkStrategy(subdomain)),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏rename this method as `getSessionReplayLink` to match `startSessionReplayRecording` wording.

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏don't set a default subdomain here (see other comment)